apps/api/src/persistence/database/arango-database-for-collection.ts

```diff
@@ -106,6 +106,15 @@ export class ArangoDatabaseForCollection<TEntity extends HasAggregateId> {
     }
 
     query(aqlQuery: AqlQuery) {
+        if (
```

Comment:
Let's remove this for now. I've added the code as a comment in [this story](https://www.pivotaltracker.com/story/show/188767831).